Dockerfile

```diff
@@ -1,40 +1,23 @@
-# Based on ruby image, packed with a lot of development dependencies
 FROM ruby:2.3
 
-# Install all dependencies for development and testing
 RUN apt-get update && \
     apt-get install --no-install-recommends -y \
-      mysql-client \
+      cron \
       libv8-dev \
       libmagic-dev \
-      libmysqlclient-dev \
-      xvfb \
-      iceweasel && \
+      libmysqlclient-dev && \
     rm -rf /var/lib/apt/lists/* && \
     apt-get clean
 
-# Run app and all commands as user 'app'. This avoids changing permissions
-# for files in mounted volume.
-RUN adduser --gecos GECOS --disabled-password --shell /bin/bash app
-USER app
```

Comment:
It's very uncommon to create a special user just for running the main entry point. Since it's a Docker container there are no "real root" rights in the container anyway and having an additional user does not offer a lot of additional security to the underlying system.